OJS.Workers.Common/Helpers/FileHelpers.cs

```diff
@@ -161,6 +161,9 @@ public static string ExtractFileFromZip(string pathToArchive, string fileName, s
 
         public static string ProcessModulePath(string path) => path.Replace('\\', '/');
 
+        public static void WriteAllText(string filePath, string text)
+            => File.WriteAllText(filePath, text);
```

Comment:
Use the service here as proposed in the other PR

Reply:
The infrastructure does not support dependency injection and service is redundant at this point.

Reply:
What about factory instantiation?

Reply:
Made a new issue: https://github.com/SoftUni-Internal/interactive-issues/issues/994